static/js/src/cookie-policy-with-callback.js

```diff
@@ -0,0 +1,35 @@
+/*
+  Set a unique user_id if the user has accepted cookies.
+  If the user has not set their cookie preference, wait for the cookie policy to run first. 
+*/
+
+import { v4 as uuidv4 } from "https://jspm.dev/uuid";
```

Comment:
I think it would be better to install and import `uuidv4` npm package instead. This import looks a bit unusual to me. Is it written somewhere that this is the way it should be done?

Reply:
It won't be available client side without a bundler, so I've used a cdn as suggested [here](https://stackoverflow.com/a/75202002)